### DISCUSSION TITLE
Wrong output when using script to paste keystrokes, only when using Rustdesk

### DISCUSSION THREAD

**Discussion by Affection4036** [rustdesk/rustdesk#8816] [Q&A] [answered]
Hello,

Another user requested a feature for [typing your keyboard as keystrokes when pasting](https://github.com/rustdesk/rustdesk/discussions/5451). While I patiently wait for this feature, I decided to find another way to achieve this. A Github user [shared their workflow](https://gist.github.com/sscotth/310db98e7c4ec74e21819806dc527e97) added within Automator (MacOS) to do this. This works flawlessly within different apps, except Rustdesk. The output pasted from clipboard seems to be different when using this script remotely and changes when changing the input type. How can I make it so it pastes correctly?

To show what's wrong with it, please find the example below. I've generated a password of 32 characters that includes normal characters (incl. captitals), numbers and special characters.

**Password in clipboard:**

- N&59kaJdJeJiF&mm65N3aWQ2bANxQa#r

**Password when pasted through Rustdesk:** 

- ![image](https://github.com/user-attachments/assets/af709f1f-e0c3-4461-b043-d8391decd7b9)

It looks like it has a problem with capital characters. Typing over passwords is a huge thing when helping several people a day. How can I solve this?

I tried every option found within Rustdesk:
![image](https://github.com/user-attachments/assets/e4dec918-1764-41e5-aa4c-9836a8d2334c)

**Comment by rustdesk** [comment, marked as answer]
It seems this feature really useful, I will add it quickly.

**Reply by Affection4036** [reply]
> It seems this feature really useful, I will add it quickly.

I am really happy about this, thanks a lot!
